src/go/util/httptest_util.go

```diff
@@ -71,3 +73,18 @@ func InitMockServerFromPathResp(pathResp map[string]string) *httptest.Server {
 		w.WriteHeader(http.StatusNotFound)
 	}))
 }
+
+// JsonEqual compares two JSON strings after normalizing them.
+func JsonEqual(x, y *string) bool {
+	normalizeJson(x)
+	normalizeJson(y)
+	return strings.EqualFold(*x, *y)
+}
+
+// NormalizeJson returns normalized JSON string.
+func normalizeJson(input *string) {
```

Comment:
can we not modify the input? Maybe there are cases that input could not be modified.

Reply:
+1

Reply:
The reason I modify the input is to make it easier for debug. 

The expected and wanted JSON string may have different order. After normalize, it is much easier to see clearly where the difference is.

Reply:
add a comments, to clearify when to use this.